src/main/java/com/github/mc1arke/sonarqube/plugin/ce/pullrequest/gitlab/GitlabServerPullRequestDecorator.java

```diff
@@ -67,10 +71,14 @@
 
 public class GitlabServerPullRequestDecorator implements PullRequestBuildStatusDecorator {
 
-    public static final String PULLREQUEST_GITLAB_URL =
-            "com.github.mc1arke.sonarqube.plugin.branch.pullrequest.gitlab.url";
-    public static final String PULLREQUEST_GITLAB_REPOSITORY_SLUG =
-            "com.github.mc1arke.sonarqube.plugin.branch.pullrequest.gitlab.repositorySlug";
+    public static final String PULLREQUEST_GITLAB_INSTANCE_URL =
+            "sonar.pullrequest.gitlab.instanceUrl";
```

Comment:
Since I don't think these properties are available in Sonarqube developer edition, could you prefix them with `com.github,mc1arke....` (or confirm that are available in Sonarqube developer edition)

Reply:
They are used by Sonarqube developer edition, but currently only internally, if the build is done in gitlab ci. They don't support other environments yet for MR decoration. https://jira.sonarsource.com/browse/MMF-1901 will fix that, and the parameters will probably get documented as soon as that is implemented.